storage/innobase/fts/fts0que.cc

```diff
@@ -953,6 +953,18 @@ fts_query_free_doc_ids(
 	query->total_size -= SIZEOF_RBT_CREATE;
 }
 
+
+/*******************************************************************//**
+Free the query intersection */
+static
+void
+fts_query_free_intersection(
+	fts_query_t*	query)		/*!< in: query instance */
```

Comment:
Either here, either in the function body the indentation is wrong (should be tabs everywhere)